webpack.config.js

```diff
@@ -0,0 +1,33 @@
+// Inspired by Furo's webpack config: https://github.com/pradyunsg/furo/blob/main/webpack.config.js
+
+const { resolve } = require("path");
```

Comment:
I'm not fully sure about this but I believe the standard with js these days is to use `import` rather than `require`. Technically I'm not sure there is a difference but generally feel like it would be good to use the latest conventions

Reply:
It's the standard for browser applications, but for webpack config it's correct to use `require`.